Dockerfile

```diff
@@ -16,6 +16,6 @@ RUN sudo apt-get update && sudo apt-get install -y \
   && sudo ldconfig \
```

Comment:
libxinerama1, libxcursor dev are not necessary IMO, also the line `sudo ldconfig` is not needed

Reply:
No worries, I just wanted you to know that this is here to use.